recipes/xsimd/run_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#conda env create -f ./test/test-environment.yml
+#source activate test-xsimd
+#mkdir build
+#cd build
+#cmake -D CMAKE_INSTALL_PREFIX=$PREFIX -D CMAKE_BUILD_TYPE=Release ..
+#make test_xsimd
+#./test_xsimd
```

Comment:
The unix version is commented because of a too old version of gcc to build the tests successfully, but this works for me.